lara-typescript/src/page-item-authoring/mw-interactives/customize.tsx

```diff
@@ -40,34 +41,67 @@ export const CustomizeMWInteractive: React.FC<Props> = (props) => {
     height: native_height,
     mode: (aspect_ratio_method || "DEFAULT") as AspectRatioMode
   });
-  const [clickToPlay, setClickToPlay] = useState(click_to_play);
-  const [enableLearnerState, setEnableLearnerState] = useState(enable_learner_state);
+
+  const [state, setState] = useState({
+    aspect_ratio_method,
+    click_to_play,
+    enable_learner_state,
+    native_height,
+    native_width,
+  });
+
+  useEffect(() => {
+    setState(currentState => ({
+      ...currentState,
+      aspect_ratio_method: aspectRatioValues.mode,
+      native_height: aspectRatioValues.height,
+      native_width: aspectRatioValues.width
+    }));
+  }, [aspectRatioValues]);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setState({
```

Comment:
I think Kirk's statement from 10 days ago is pertinent here.  We should probably use the functional form to update the state:

``` 
 setState(state => ({ ...state, ... etc })
```

I wasn't aware of this issue myself, so I am going to have to do a small code audit on the work I have been doing.

Here is a blog post that explains more: https://blog.logrocket.com/a-guide-to-usestate-in-react-ecb9952e406c/
> If you use the previous value to update state, you must pass a function that receives the previous value and returns the new value: